content/tutorials/use-simple-xrp-payments/send-xrp.md

```diff
@@ -0,0 +1,593 @@
+# Send XRP
+
+This tutorial walks through send a simple XRP Payment using JavaScript, by first stepping through the process using the XRP Test Net, then comparing that to the additional requirements for doing the equivalent in production.
```

Comment:
* Missing "how to" or something like it
* Is the comma right? Seems wrong. Not sure.
* Also, not to edit for style but: "through ... through" and "using ... using"

Maybe:
> This tutorial explains how to send a simple XRP Payment using JavaScript by stepping through the process with the XRP Test Net, then comparing that to the additional requirements for doing the equivalent in production.